### PR TITLE
test: Manually truncate path string

### LINKED ISSUE #14401: `integration::run::worker_event_handler_test` breaks for long paths to the Deno checkout
If the Deno checkout is at `/some/particular/way/too/overly/long/path/to/deno`, the test's output is:
```
Target from self.onmessage: [object DedicatedWorkerGlobalScope]
Target from message event listener: [object DedicatedWorkerGlobalScope]
Arguments from self.onerror: [
  "Uncaught Error: Some error message",
  "file:///some/particular/way/too/overly/long/path/to/deno/cli/tests/testdata/workers/worker_event_han...",
  9,
  9,
  Error: Some error message
    at file:///some/particular/way/too/overly/long/path/to/deno/cli/tests/testdata/workers/worker_event_handlers.js:9:9
    at innerInvokeEventListeners (deno:ext/web/02_event.js:754:9)
    at invokeEventListeners (deno:ext/web/02_event.js:794:7)
    at dispatch (deno:ext/web/02_event.js:663:11)
    at dispatchEvent (deno:ext/web/02_event.js:1006:14)
    at pollForMessages (deno:runtime/js/99_main.js:153:9)
]
Is event canceled?: true
```

but the expectation is:
```
Target from self.onmessage: [object DedicatedWorkerGlobalScope]
Target from message event listener: [object DedicatedWorkerGlobalScope]
Arguments from self.onerror: [
  "Uncaught Error: Some error message",
  "[WILDCARD]/worker_event_handlers.js",
  9,
  9,
  Error: Some error message
    at [WILDCARD]
]
Is event canceled?: true
```

The console formatting cuts off the end of the path, breaking the expectation.

### PR DESCRIPTION
**Pull request by nayeemrmn**
Fixes #14401.

### CODE REVIEW

**Review by bartlomieju — CHANGES_REQUESTED** [review]
Can we get a test for this?

**Comment by nayeemrmn** [comment]
> Can we get a test for this?

No, it's specific to how each test is written and occurrence depends on how nested the contributor's deno repo is in their filesystem. Happened back in https://github.com/denoland/deno/pull/14159#pullrequestreview-935684183 as well. There are probably more out there, no general solution.

**Comment by bartlomieju** [comment]
> > Can we get a test for this?
> 
> No, it's specific to how each test is written and occurrence depends on how nested the contributor's deno repo is in their filesystem. Happened back in [#14159 (review)](https://github.com/denoland/deno/pull/14159#pullrequestreview-935684183) as well. There are probably more out there, no general solution.

Maybe we could create a nested directory structure in `cli/tests/testdata` that is over 200 chars long to showcase it?

**Comment by nayeemrmn** [comment]
> Maybe we could create a nested directory structure in `cli/tests/testdata` that is over 200 chars long to showcase it?

But we'd only be testing it and applying the fix for that test... I don't think there's a point.

File: cli/tests/testdata/workers/worker_event_handlers.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ self.addEventListener("message", (evt) => {
 });
 
 self.onerror = (...args) => {
+  args = args.map((v) => typeof v == "string" ? v.slice(-100) : v);
```
Comment (dsherret) [inline cli/tests/testdata/workers/worker_event_handlers.js:13]: Could use a comment explaining why this is done.

Comment (bartlomieju) [reply cli/tests/testdata/workers/worker_event_handlers.js:13]: @nayeemrmn please add a comment so we can land this PR

**Review by bartlomieju — APPROVED** [review]
LGTM

**Pull request merged**